8by8-org/web-app
# Save voter registration data in Firebase

## Overview 
> REQUIRED
> Fill in the overview of this PR, what this PR is trying to achieve
Save voter registration data in Firebase and add a link to generate pdf again on the /voterreg/completed page
#159 



## Test Plan 
> REQUIRED 
- Create a new user
- Go to /voterreg and fill in the info
- Once completed, check email to see if a form is received.
- Check firebase to see if voteInfo is saved
- Click the "Get email with PDF file" and check email to see if the same form is received
- Refresh the page and click the "Get email with PDF file" and check email to see if the same form is received
- log out and log in again and click the "Get email with PDF file" and check email to see if the same form is received

## Follow ups 
> It is okay for the PR to be not perfect, list what you/other should work on after this PR is merged
